lib/fog/azurerm/models/network/subnets.rb

```diff
@@ -20,8 +20,10 @@ def all
           load(subnets)
         end
 
-        def get(identity)
-          all.find { |f| f.name == identity }
+        def get(resource_group, virtual_network_name, subnet_name)
```

Comment:
Actually I am following fog-azure-rm convention which is to put the container on the most left.